packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -131,7 +131,7 @@ describe.each([undefined, 400])('applyNeverFilter - timeout: %s', timeout => {
   });
 
   beforeEach(() => {
-    makeNeverFilter = jest.spyOn(TableUtils, 'makeNeverFilter');
+    makeNeverFilter = jest.spyOn(TableUtils.prototype, 'makeNeverFilter');
```

Comment:
Probably better to just spy on the `tableUtils` instance/mockReturnValue in a `beforeAll` block, and then just do a `mockClear` in `beforeEach`